src/lib/ngsiNotify/senderThread.h

```diff
@@ -2,47 +2,63 @@
 #define SENDERTHREAD_H
```

Comment:
Which is the relation between adding Format to  SenderThreadParams struct and the cache work? Just to understand...